### PR TITLE
add solution and tests for ex 9 delete comment

### PR DESCRIPTION
**Pull request by drewanderson201**
add solution and tests for ex 9 delete comment

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -40,6 +41,9 @@ app.post("/api/articles/:article_id/comments", postComment)
 
 app.patch("/api/articles/:article_id", patchArticle);
 
+app.delete("/api/comments/:comment_id", deleteComment);
+
+
```
Comment (nc-niamh-smith) [inline app.js:47]: Where do you refer to your error handlers?

**Review by nc-niamh-smith — COMMENTED** [review]
🙌 

I'm a bit confused about your app.js - where do you refer to your error handlers?

File: app.js (hunk 1 above)
Comment (drewanderson201) [reply app.js:47]: Hi @nc-niamh-smith do you mean where is the file? There is a folder called errors with an index.js file with all error handler function in it.

Comment (nc-niamh-smith) [reply app.js:47]: No I understand that - I'm unsure as to where you are referring to it following the MVC model? I'm sure it all works correctly, but usually you would have a reference inside an app.use, for example: app.use(errorHandler). I'm just wondering how your errors make it to the separate file?

**Pull request merged**